changelog.txt

```diff
@@ -1,3 +1,14 @@
+Version 3.0.1
+-------------
+- Adding dual screen support for Surface Duo.
+  * Please exclude classes under "com.microsoft.device.display" in your proguard file.
+  * For apps that would like to include their own copy of Surface Duo SDK, please exclude the SDK inside ADAL as follows.
```

Comment:
I think this needs to be front & center in the readme -- we did the same thing for Gson here:
https://github.com/AzureAD/azure-activedirectory-library-for-android#option-2-binaries-via-gradle